client/fcm.js

```diff
@@ -0,0 +1,41 @@
+export async function initializeFCM(messageSenderId) {
+  if ( !messageSenderId ){
+      console.log("Cannot subscribe to notifications at this moment");
+      return;
+  }
+  Promise.all([
+    import (/* webpackChunkName: "firebase-app" */ "firebase/app"),
+    import(/* webpackChunkName: "firebase-messaging" */ "firebase/messaging")
+  ]).then(async ([ firebase, m ]) => {
+    try {
+        firebase.initializeApp({
+          messagingSenderId: messageSenderId.toString()
```

Comment:
what is `toString` for in this case?

Reply:
messageSenderId though is a number, should be provided only as a string to intializeApp. Just a safety check so that if the messageSender Id is provided as a number in publisher config it will be converted to a string.